tools/challenge-parser/parser/index.js

```diff
@@ -50,9 +50,10 @@ exports.parseMD = function parseMD(filename) {
       if (err) {
         err.message += ' in file ' + filename;
         reject(err);
+      } else {
+        delete file.contents;
+        resolve(file.data);
       }
```

Comment:
```suggestion
      if (!err) {
        delete file.contents;
        resolve(file.data);
      } 

      err.message += ' in file ' + filename;
      reject(err);
```

Is this more readable?